Library/Homebrew/test/utils/github_spec.rb

```diff
@@ -25,9 +25,10 @@
       results = described_class.search_issues("brew search",
                                               repo:   "Homebrew/legacy-homebrew",
                                               author: "MikeMcQuaid",
-                                              is:     "closed")
+                                              is:     "issue",
```

Comment:
For some reason, `is: closed` is giving me multiple results which doesn't match manual GitHub search,
`["Shall we move more things to taps?", "upgrade: announce usage of --all.", "Remove (almost) everything", "bin/brew: move logic to Library.", "install: search when formula isn't found.", "Remove MD5 support.", "config: fail when HOMEBREW_BREW_FILE is unset.", "Use `(installed)` and emoji ticks consistently.", "Import docs from wiki."]`

Currently, filtering on `is: issue` and `no: milestone` gives one result.